gcb/auth_logic.sh

```diff
@@ -0,0 +1,3 @@
+AUTH_LOGIC_DIR="rust/tools/authorization-logic"
```

Comment:
Add the following to the beginning? Add at least `set -e` so that the script exits on error.
```
#!/bin/bash
#
set -e
```